src/Search/SearchItem.tsx

```diff
@@ -27,10 +40,38 @@ export function SearchItem({ item }: SearchItemProps) {
   }, [item.active]);
 
   return (
-    <SearchLink to={item.url} tabIndex={0} innerRef={ref} data-component-name="Search/SearchItem">
-      <Title>{item.title}</Title>
-      <Description>{highlight(item.chunks)}</Description>
-      <Path>{item.path.join(' > ')}</Path>
+    <SearchLink to={item.url} tabIndex={0} innerRef={ref}>
+      <Operaition>
+        {item.httpVerb ? <OperationBadge type={item.httpVerb}>{item.httpVerb}</OperationBadge> : null}
+        {highlight(item.pathName)}
+      </Operaition>
```

Comment:
I'd hide operation for regular text results (Mardkdown):
```suggestion
      {item.httpVerb ? <Operaition>
          <OperationBadge type={item.httpVerb}>{item.httpVerb}</OperationBadge> 
          {highlight(item.pathName)}
        </Operaition>
      : null}
```